src/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ ELSEIF (UNIX)
     ELSEIF (CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
         SET(LIBS_SYSTEM compat pthread)
     ELSE()
-        SET(LIBS_SYSTEM c pthread)
```

Comment:
Please verify what is the value of CMAKE_SYSTEM_NAME and Platform when building for Android with and without Conan, so that we can update accordingly.

Reply:
handled